Sources/Plasma/NucleusLib/pnKeyedObject/plUoid.h

```diff
@@ -158,6 +163,15 @@ class plUoid
     plUoid(const plLocation& location, uint16_t classType, const ST::string& objectName, const plLoadMask& m=plLoadMask::kAlways);
     plUoid(plFixedKeyId fixedKey);
     plUoid(const plUoid& src);
+    plUoid(plUoid&& move)
+        : fObjectID(move.fObjectID), fClonePlayerID(move.fClonePlayerID),
+          fCloneID(move.fCloneID), fClassType(move.fClassType),
+          fObjectName(std::move(move.fObjectName)),
+          fLocation(move.fLocation), fLoadMask(move.fLoadMask)
+    {
+        move.Invalidate();
```

Comment:
Do we actually need to invalidate the other object?  The only member that matters (`fObjectName`) is already taken care of by the move above, and the destructor calls `Invalidate()` anyway (probably also not necessary, TBH).

In fact, this whole move constructor could probably be `= default`.

Reply:
I was on the fence about this myself - I was thinking that by invalidating the other object, it would be clearer that a move had taken place. Later, I added the bugprone-use-after-move check, so the invalidation is probably redundant.